tutor/specs/e2e/course-settings.e2e.ts

```diff
@@ -83,6 +81,7 @@ describe('Course Settings', () => {
             await page.click('testEl=delete-course-btn')
             await expect(page).toHaveSelector('testEl=disabled-delete-course-message-warning')
             await expect(page).toHaveText('testEl=delete-course-message', 'leave')
+	    await page.click('button.close')
```

Comment:
also alignment?

Reply:
Something with my Emacs config on Windows prevented me from seeing this :( I'll fix 'em